kclvm/api/src/service/service.rs

```diff
@@ -0,0 +1,72 @@
+use std::{path::Path, string::String, time::SystemTime};
+
+use crate::model::gpyrpc::*;
+
+use kclvm_parser::load_program;
+use protobuf_json_mapping::print_to_string_with_options;
+use protobuf_json_mapping::PrintOptions;
+
+#[derive(Default)]
+pub struct KclvmService {}
+
+impl KclvmService {
+    pub fn ping(&self, args: &Ping_Args) -> Ping_Result {
+        Ping_Result {
+            value: (args.value.clone()),
+            special_fields: (args.special_fields.clone()),
+        }
+    }
+
+    pub fn exec_program(&self, args: &ExecProgram_Args) -> ExecProgram_Result {
```

Comment:
`exec_program` API and CLI implementation and error handling need to be considered uniformly,